specification/introduction/introduction.tex

```diff
@@ -64,31 +83,30 @@ \section{Maintenance of the standard data type set}
 forum\footnote{Please refer to \href{http://uavcan.org}{uavcan.org}.}
 which is open to the general public.
 
-The set of standard data type definitions\footnote{The Data Structure Description Language (DSDL) and related concepts are described in section \ref{sec:dsdl}.}
-is an integral part of the specification, however, there is only a small set of required data types needed to implement the protocol.
+The set of standard data type definitions\footnote{The \emph{data structure description language} (DSDL) and
+related concepts are described in section \ref{sec:dsdl}.} is an integral part of the specification;
+however, there is only a small set of required data types needed to implement the protocol.
 A larger set of optional data types are defined to create a standardized data exchange environment
-supporting the interoperability of COTS\footnote{Commercial off-the-shelf equipment.} equipment manufactured by different vendors.
+supporting the interoperability of COTS\footnote{Commercial off-the-shelf equipment.}
+equipment manufactured by different vendors.
 See chapter \ref{sec:application_layer} for more information.
 
-Within the same major version, the set of standard data type definitions can be modified only in the following ways:
+Within the same major version of the specification,
```

Comment:
Should we add a similar section for what modifications are allowed within a minor version?

Reply:
Let us finish the DSDL chapter first, and then we'll consider whether the whole section "Maintenance of the standard data type set" should be moved into the DSDL chapter. For now I suggest to keep it as-is.